python/tk_multi_loader/dialog.py

```diff
@@ -185,6 +210,23 @@ def __init__(self, action_manager, parent=None):
         self.ui.publish_view.setContextMenuPolicy(QtCore.Qt.ActionsContextMenu)
 
         #################################################
+        # popdown publish filter widget for the main view
```

Comment:
Cleanup of this block since you kept our version of the search box ? 

Also "_on_publish_filter_clicked" (line 437) isn't used anywhere since this is commented and could be removed...